backend/orders/views.py

```
@@ -1,3 +1,20 @@
-from django.shortcuts import render
+from orders.serializers import (
+    OrderItemSerializer,
```

Ruff (0.12.2)

`orders.serializers.OrderItemSerializer` imported but unused

Remove unused import

(F401)